lib/Data/Fake/Text.pm

```diff
@@ -33,6 +33,7 @@ provide the number of words); the default is one.
 
 sub fake_words {
     my ($count) = @_;
+    $count = 1 unless defined $count;
```

Comment:
I avoided using `//=` here because it would bump the required Perl version to 5.10.0